app/controllers/api/v1.rb

```diff
@@ -1,4 +1,9 @@
 module Api
   module V1
+    class RootController < ApplicationController
```

Comment:
This file is identical to `app/controller/api/v1/root_contoller.rb` except for the path. How does it work? Why here  `Api::Doc["1.0"]` and your actual test for `v1/openapi.json` returns 1.1?

Reply:
If `v1` is passed in as a route it never gets to this file. The redirect here calls the latest minor version listed here: https://github.com/RedHatInsights/approval-api/blob/master/config/routes.rb#L13

Reply:
Though to your point. This file will go away in a followup PR so that should also clear up the redundancy.